src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_utils.cpp

```diff
@@ -59,16 +59,19 @@ BrgemmConfig::BrgemmConfig(const dnnl::impl::cpu::x64::cpu_isa_t& isa,
       m_with_compensations(src_dt == ov::element::i8 && !one_of(m_isa, avx512_core_amx, avx2_vnni_2)),
       m_are_wei_constant(are_wei_constant),
       m_transposed_b(transposed_b),
-      m_are_wei_blocked(ov::intel_cpu::pass::BrgemmCPUBlocking::is_kn_blocking_supported(src_dt) && m_are_wei_constant),
+      m_are_wei_blocked(ov::intel_cpu::pass::BrgemmCPUBlocking::is_kn_blocking_supported(src_dt)),
       m_wei_k_blk(get_elems_in_vec(wei_dt)) {
     const auto is_fp32 = src_dt == ov::element::f32 && wei_dt == ov::element::f32;
 
     // FC always requires weight repacking
+    // BrgemmToBrgemmCPU insert BrgemmCopyB based on this flag.
+    // EliminateBrgemmCopyB remove BrgemmCopyB to outside and markup runtime config
+    // BrgemmExternalRepackingAdjuster set runtime executor for runtime config
```

Comment:
Could you tell me please if we really need to have this comment?
If it's so, could you please a little bit rephrase the text? To be honest I don't get it :(

Reply:
I think the passes name and description in passes header file are self explainable, so removed this additional comments.